clio-client/src/main/scala/org/broadinstitute/clio/client/util/ClassUtil.scala

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.clio.client.util
+
+object ClassUtil {
```

Comment:
Extreme nitpick: this is generic enough that it might belong in the clio-util project.